app/views/ctc/questions/confirm_payment/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<% @main_title = @not_collecting ?  t("views.ctc.questions.confirm_payment.client_not_collecting") : t("views.ctc.questions.confirm_payment.title") %>
+<% content_for :page_title, @main_title %>
+
+<% content_for :form_card do %>
+  <h1 class="h2"><%= @main_title %></h1>
+
+  <div class="review-box">
+    <div class="spacing-above-0">
+      <p class="spacing-below-15"><strong><%= t('views.ctc.questions.confirm_payment.ctc_due') %></strong></p>
+      <p class="spacing-below-15"><%= number_to_currency(@ctc_amount, precision: 0, locale: :en) %></p>
+      <%= link_to t('views.ctc.questions.confirm_payment.ctc_due_link'), questions_confirm_dependents_path, class: "link" unless @ctc_amount.zero? %>
+      <%= link_to t('views.ctc.questions.confirm_payment.ctc_0_due_link'), Ctc::Questions::Dependents::InfoController.to_path_helper(id: current_intake.new_dependent_token), class: "link" if @ctc_amount.zero? %>
+    </div>
+
+    <div class="spacing-above-35">
+      <p class="spacing-below-15"><strong><%= t('views.ctc.questions.confirm_payment.rrc') %></strong></p>
+      <p class="spacing-below-15"><%= calculated_or_provided_dollar_amount(@rrc_amount) %></p>
+      <%= link_to (@rrc_amount.zero? ? t('views.ctc.questions.confirm_payment.edit_rrc_link') : t('views.ctc.questions.confirm_payment.rrc_link')), questions_stimulus_owed_path, class: "link" %>
+    </div>
+
+    <div class="spacing-above-35">
+      <p class="spacing-below-15"><strong><%= t('views.ctc.questions.confirm_payment.third_stimulus') %></strong></p>
+      <p class="spacing-below-15"><%= number_to_currency(@third_stimulus_amount, precision: 0, locale: :en) %></p>
+      <p class="spacing-below-15"><i><%= t('views.ctc.questions.confirm_payment.third_stimulus_help_text') %></i></p>
+      <p><%= t('views.ctc.questions.confirm_payment.third_stimulus_link_html') %></p>
+    </div>
+  </div>
+
+  <%= link_to t("general.continue"), next_path, class: "button button--primary button--wide text--centered spacing-above-60" unless @not_collecting%>
+  <%= link_to t('views.ctc.questions.confirm_payment.do_not_file'), root_path, class: "button button--primary button--wide text--centered spacing-above-60" if @not_collecting %>
```

Comment:
In cases where the client selects do not file here, I think we might want to do a bit more to their intake object and in the interface to tell them whats happening. For example, this will interrupt their intake, but I think that it wont remove the current_intake id from the session object. 

When clicking this button, I think we should consider:
- Moving the tax return object to "not filing" 
- Flashing a flash notice telling the client "thanks for using GetCTC, we won't be filing your return" or some such
- Remove the intake_id from the current session

Reply:
> Moving the tax return object to "not filing"

This is a good idea. The story says we should "archive the client," but we don't have archiving the client yet. I asked Anu, and she and and Kelly will chat with us to advise.

> Flashing a flash notice telling the client "thanks for using GetCTC, we won't be filing your return" or some such

Might be a good idea. We can raise to Anu/Kelly.

> Remove the intake_id from the current session

Seems very wise; might be good to fully destroy the session; this is basically sign out.